build.gradle.kts

```
@@ -66,6 +67,14 @@ subprojects {
       exceptionFormat = TestExceptionFormat.FULL
     }
   }
+
+  tasks.withType<KotlinCompile<*>> {
+    kotlinOptions {
+      freeCompilerArgs = listOf(
```

```suggestion
      freeCompilerArgs += listOf(
```
Don't clobber existing options